docs/terminology.md

```diff
@@ -33,3 +34,4 @@ The Valhalla organization is comprised of several repositories each responsible
 * tiled routing - method of building a path on graph data that has been split into square cells.
 * time - the number of seconds estimated to complete a maneuver or trip, including any additional costs.
 * trip - results of an entire route, including locations, legs, and maneuvers.
+* elevation - computing the height and range for a series of lat/lng pairs of a line or shape.
```

Comment:
This should be the definition of elevation profile. I would define elevation as the height above sea level at a specific location (lat,lng). Then have what is currently here as a definition of elevation profile and perhaps mention it is useful for charting/graphing.